src/rgw/rgw_sal_motr.cc

```diff
@@ -275,7 +275,7 @@ int MotrUser::create_bucket(const DoutPrefixProvider* dpp,
     // Create a new bucket: (1) Add a key/value pair in the
     // bucket instance index. (2) Create a new bucket index.
     MotrBucket* mbucket = static_cast<MotrBucket*>(bucket.get());
-    ret = mbucket->put_info(dpp, y, ceph::real_time())? :
+    ret = mbucket->put_info(dpp, true, ceph::real_time())? :
```

Comment:
Add comment as below:
"put_info" accepts boolean value mentioning whether to create new or update existing. "yield" is not a boolean flag, hence explicitly passing true, meaning create a new record.

Reply:
Done